src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -9,7 +9,9 @@
     public static final String MESSAGE_INVALID_ENTRY_INDEX = "The entry index provided is invalid";
     public static final String MESSAGE_DELETE_ENTRY_SUCCESS = "Deleted Entry: %1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
-    public static final String MESSAGE_INVALID_DATE_RANGE = "The provided start date is after the provided end date!";
+    public static final String MESSAGE_INVALID_DATE_RANGE = "The provided start date or time is after the provided"
+            + " end date or time!";
+    public static final String MESSAGE_DATE_IN_PAST = "Time travel is not allowed! Please provide a valid date time!";
```

Comment:
let's change this to MESSAGE_ENTRY_START_DATE_IN_PAST

Reply:
also use "end date and time", "or time" is kinda weird

Reply:
Is it ok to stick with or time? Cause i wanna try to indicate that it's either date or time that is not a valid range. Could be that start time is after end time or start date is after end date, or is there a better phrasing?

Reply:
heh, i look at it like "date and time" is a phrase. anything ig

Reply:
Ok i changed it a little, see if it's less weird like that!